expvar/expvar_test.go

```diff
@@ -0,0 +1,238 @@
+package expvar
+
+import (
+	"context"
+	"crypto/rand"
+	"fmt"
+	"math/big"
+	"reflect"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/asecurityteam/settings"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConfigName(t *testing.T) {
+	assert.Equal(t, "expvar", (&Config{}).Name())
+}
+
+func TestConfigDescription(t *testing.T) {
+	assert.Equal(t, "Expvar metric names", (&Config{}).Description())
+}
+
+func TestReport(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	randGen, err := rand.Int(rand.Reader, big.NewInt(1000))
```

Comment:
Changed from original file to address linting issue.

> G404: Use of weak random number generator (math/rand instead of crypto/rand) (gosec)
>         randGen := rand.New(rand.NewSource(time.Now().UnixNano()))

Reply:
Appreciate you fixing these! I can backport this test fix into the original package, I think we temporarily had added some TODO comments to fix these later